### ISSUE TITLE
Uncaught TypeError: Cannot read property 'definition' of null

### ISSUE THREAD

**Issue by kripod** [mulesoft/api-workbench#81]
Just use the following RAML file to reproduce:

#%RAML 1.0 DataType
type: object
properties:
  id: integer
  username: string
example:
  id: 0
  username: Server

**Atom Version**: 1.3.2
**System**: Microsoft Windows 10 Home
**Thrown From**: [api-workbench](https://github.com/mulesoft/api-workbench) package, v0.8.9

### Stack Trace

Uncaught TypeError: Cannot read property 'definition' of null

```
At C:\Users\user\.atom\packages\api-workbench\main.js:8

TypeError: Cannot read property 'definition' of null
    at Object.i [as getType] (C:\Users\user\.atom\packages\api-workbench\main.js:8:4348)
    at C:\Users\user\.atom\packages\api-workbench\main.js:7:30470
    at Array.forEach (native)
    at Object.a [as typeFromNode] (C:\Users\user\.atom\packages\api-workbench\main.js:7:29998)
    at Object.d [as findExampleContentType] (C:\Users\user\.atom\packages\api-workbench\main.js:8:18142)
    at l (C:\Users\user\.atom\packages\api-workbench\main.js:30:10208)
    at f (C:\Users\user\.atom\packages\api-workbench\main.js:30:14193)
    at Object.m [as getSuggestions] (C:\Users\user\.atom\packages\api-workbench\main.js:30:12197)
    at C:\Users\user\AppData\Local\atom\app-1.3.2\resources\app.asar\node_modules\autocomplete-plus\lib\autocomplete-manager.js:286:56
    at Array.forEach (native)
    at AutocompleteManager.module.exports.AutocompleteManager.getSuggestionsFromProviders (C:\Users\user\AppData\Local\atom\app-1.3.2\resources\app.asar\node_modules\autocomplete-plus\lib\autocomplete-manager.js:265:17)
    at AutocompleteManager.getSuggestionsFromProviders (C:\Users\user\AppData\Local\atom\app-1.3.2\resources\app.asar\node_modules\autocomplete-plus\lib\autocomplete-manager.js:3:61)
    at AutocompleteManager.module.exports.AutocompleteManager.findSuggestions (C:\Users\user\AppData\Local\atom\app-1.3.2\resources\app.asar\node_modules\autocomplete-plus\lib\autocomplete-manager.js:252:19)
    at C:\Users\user\AppData\Local\atom\app-1.3.2\resources\app.asar\node_modules\autocomplete-plus\lib\autocomplete-manager.js:3:61
```

### Commands

```
     -0:31.2.0 editor:fold-selection (atom-text-editor.editor.is-focused)
     -0:28.5.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:27.7.0 editor:move-to-previous-subword-boundary (atom-text-editor.editor.is-focused)
     -0:18 autocomplete-plus:confirm (atom-text-editor.editor.is-focused.autocomplete-active)
  2x -0:12.4.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:04.6.0 core:save (atom-text-editor.editor.is-focused)
     -0:03.7.0 editor:newline (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
api-workbench, v0.8.9
keyboard-localization, v1.4.11
linter, v1.11.3

# Dev
No dev packages
```

**Comment by ddenisenko** [comment]
Reproducible. Thank you for reporting the bug.

**Comment by ddenisenko** [comment]
Fixed. Fix should be available in 0.8.10.
